responses/pl/HassTurnOff.yaml

```diff
@@ -2,8 +2,9 @@ language: pl
 responses:
   intents:
     HassTurnOff:
-      default: Turned off {{ slots.name }}
-      lights_area: Turned off lights in {{ slots.area }}
-      fans_area: Turned off fans in {{ slots.area }}
-      cover: Closed {{ slots.name }}
-      cover_area: Closed {{ slots.area }}
+      default: Wyłączono {{ slots.name }}
+      lights_area: Wyłączono lampy w {{ slots.area }}
```

Comment:
Maybe "Wyłączono **światło** w [salonie]"? What is your opinion?

Reply:
Good point 💪🏻